pcweb/components/footer.py

```diff
@@ -18,7 +18,7 @@
     "font_family": styles.SANS,
     "font_weight": "500",
     "_hover": {"color": "#82799E"},
-    "color": "#AA9EC3",
+    "color": "#6C6C81"
```

Comment:
For all the colors, instead of the hex codes could we using the radix colors that are close to it? Like for this it could be `rx.color("gray", 9)` where our gray color is "mauve". Then we could stick to using our own color system everywhere, and potentially even only use gray/accent in many places esp. for the text colors.